client/landing/subscriptions/components/site-subscriptions-list/site-subscriptions-list-actions-bar.tsx

```diff
@@ -30,10 +28,11 @@ const ListActionsBar = () => {
 	return (
 		<div className="site-subscriptions-list-actions-bar">
 			<SearchInput
+				delaySearch
 				placeholder={ translate( 'Search by site name or address…' ) }
 				searchIcon={ <SearchIcon size={ 18 } /> }
 				onSearch={ setSearchTerm }
-				defaultValue={ initialUrlQuerySearchTerm }
+				defaultValue={ searchTerm }
```

Comment:
The default value of the search input is simply `searchTerm`: its initial value, because subsequent value changes are not used. The initial value is set in `SiteSubscriptionsQueryPropsProvider` where it's read from the query params. Here we don't need to touch the query params no longer.